### PR TITLE
Migrate all imports/references to OpenFaaS org

### PR DESCRIPTION
**Pull request by johnmccabe**
## Description

This commit updates all imports and references in documentation/scripts to point at the new github OpenFaaS org.

## Motivation and Context

- [x] I have raised an issue to propose this change (#116)

## How Has This Been Tested?

Have run `build_redist.sh` to regenerate clients on windows. Have checked the windows client behaves as before.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/faas/blob/master/CONTRIBUTING.md) guide
- [ ] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by johnmccabe** [comment]
@alexellis I just want to check this on the Mac as well, have been working on Windows 10 the past few days

**Comment by johnmccabe** [comment]
This is also related to https://github.com/openfaas/faas/issues/123

File: sample/github_hmac/Dockerfile [hunk 1]
```diff
@@ -2,13 +2,13 @@ FROM golang:1.7.5-alpine
 
 MAINTAINER alexellis2@gmail.com
 ENTRYPOINT []
-#RUN mkdir -p /go/src/github.com/alexellis/faas/sample-functions/handler
-WORKDIR /go/src/github.com/alexellis/faas/sample-functions/handler
-COPY . /go/src/github.com/alexellis/faas/sample-functions/handler
+#RUN mkdir -p /go/src/github.com/openfaas/faas/sample-functions/handler
```
Comment (alexellis) [inline sample/github_hmac/Dockerfile]: Is there an additional change here?

**Comment by alexellis** [comment]
@johnmccabe when you're happy I'll merge

File: sample/github_hmac/Dockerfile (hunk 1 above)
Comment (johnmccabe) [reply sample/github_hmac/Dockerfile]: do you mean the commented out line? no that was already there

**Comment by johnmccabe** [comment]
@alexellis this should be good to merge

fyi @ilovezfs

**Comment by ilovezfs** [comment]
Nice!

**Pull request merged**